<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacesWidget</class>
 <widget class="QWidget" name="InterfacesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QTableWidget" name="m_outputTable">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Output Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Width</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="m_addInput">
     <property name="text">
      <string>Add Input...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="m_addOutput">
     <property name="text">
      <string>Add Output...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="m_removeOutput">
     <property name="text">
      <string>Remove Output</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="m_removeInput">
     <property name="text">
      <string>Remove Input</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="m_inputTable">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Input Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Width</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
